src/Generator/HtmlForPdfGenerator.php

```diff
@@ -14,39 +13,30 @@ class HtmlForPdfGenerator
 {
     use GeneratorTrait;
 
-    /** @var Environment[] */
-    private $environments;
-
-    public function __construct(array $documents)
-    {
-        $this->environments = array_map(
-            function (DocumentNode $document) {
-                return $document->getEnvironment();
-            },
-            $documents
-        );
-    }
+    public function generateHtmlForPdf(
```

Comment:
When array is the type-hint, we should add an `@param` phpdoc with what it actually is - e.g. `Document[]`